addons/iap/models/iap_account.py

```diff
@@ -19,9 +20,95 @@ class IapAccount(models.Model):
     _rec_name = 'service_name'
     _description = 'IAP Account'
 
-    service_name = fields.Char()
-    account_token = fields.Char(default=lambda s: uuid.uuid4().hex)
+    service_name = fields.Char(string='Account Name')
+    account_token = fields.Char(
+        default=lambda s: uuid.uuid4().hex,
+        help="Account token is your authentication key for this service. Do not share it.",
+        size=43)
     company_ids = fields.Many2many('res.company')
+    account_info_id = fields.Many2one(
+        'iap.account.info', compute='_compute_info', inverse='_inverse_info', search='_search_info')
+    account_info_ids = fields.One2many(
+        'iap.account.info', 'account_id',
+        invisible=True, string="Accounts from IAP")
+    iap_service_name = fields.Char(string='Related Service', related='account_info_id.service_name')
+    balance = fields.Char(compute='_compute_balance')
+    description = fields.Char(related='account_info_id.description')
+    warn_me = fields.Boolean(
+        related='account_info_id.warn_me',
+        help="We will send you an email when your balance gets below that threshold",
+        readonly=False)
+    warning_threshold = fields.Float(related='account_info_id.warning_threshold', readonly=False)
+    warning_email = fields.Char(related='account_info_id.warning_email', readonly=False)
+    show_token = fields.Boolean(default=False)
+
+    @api.model
+    def get_view(self, view_id=None, view_type='form', **options):
+        res = super().get_view(view_id, view_type, **options)
```

Comment:
The keyword parameters should be passed as keyword parameters.
Even better, since you're only interested in the value of `view_type`, you could do it this way : 
```py
    def get_view(self, *args, **kwargs):
        res = super().get_view(*args, **kwargs)
        if kwargs['view_type'] == 'tree':
        (...)
```
This way, any change in the signature of `get_view` won't break this, unless, of course, it's a change regarding the `view_type` argument.